packages/beacon-node/src/api/impl/beacon/blocks/index.ts

```diff
@@ -550,3 +567,14 @@ async function reconstructBuilderBlockOrContents(
   const signedBlockOrContents = await executionBuilder.submitBlindedBlock(signedBlindedBlock);
   return signedBlockOrContents;
 }
+
+async function delegateBlindedBlockToBuilder(
```

Comment:
I don't like the term delegate here

maybe just
```suggestion
async function submitBlindedBlockToBuilder(
```